package.json

```diff
@@ -33,116 +33,112 @@
     "postshrinkwrap": "rewrite-lockfile package-lock.json"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
-    "@mdi/font": "^3.7.95",
-    "auto-bind": "^2.1.0",
-    "aws-sdk": "^2.342.0",
+    "@mdi/font": "^4.5.95",
+    "auto-bind": "^2.1.1",
+    "aws-sdk": "^2.547.0",
     "axios": "^0.19.0",
     "bcryptjs": "^2.3.0",
-    "blob-util": "^1.2.1",
-    "bluebird": "^3.4.6",
-    "body-parser": "^1.15.2",
-    "bootstrap-sass": "^3.3.7",
+    "blob-util": "^2.0.2",
+    "bluebird": "^3.7.0",
+    "body-parser": "^1.19.0",
+    "bootstrap-sass": "^3.4.1",
     "bootswatch": "^3.3.7",
-    "boxen": "^4.0.0",
+    "boxen": "^4.1.0",
     "bunyan": "^1.8.5",
+    "core-js": "^3.2.1",
     "cors": "^2.8.1",
-    "cropperjs": "^1.0.0-beta.1",
-    "cuid": "^1.3.8",
+    "cropperjs": "^1.5.6",
+    "cuid": "^2.1.6",
     "d3-hierarchy": "^1.1.5",
     "d3-selection": "^1.2.0",
     "d3-shape": "^1.2.0",
-    "d3-zoom": "^1.7.1",
+    "d3-zoom": "^1.8.3",
     "dom-shims": "^0.1.4",
-    "dotenv": "^2.0.0",
-    "email-regex": "^3.0.0",
+    "dotenv": "^8.1.0",
     "emailjs": "^2.0.0",
-    "event-source-polyfill": "^0.0.12",
-    "exceljs": "^1.6.0",
-    "express": "^4.14.0",
+    "event-source-polyfill": "^1.0.9",
+    "exceljs": "^2.0.1",
+    "express": "^4.17.1",
     "express-async-errors": "^3.1.1",
-    "fecha": "^2.3.0",
-    "file-type": "^7.5.0",
-    "focus-trap": "^2.3.1",
-    "hash-obj": "^1.0.0",
+    "fecha": "^3.0.3",
+    "focus-trap": "^5.0.2",
+    "hash-obj": "^2.0.0",
```

Comment:
This probably shouldn't be bumped to latest version due to:
https://github.com/sindresorhus/hash-obj/releases/tag/v3.0.0
> It now sorts object keys of objects inside arrays 6d0424f
This may change the hash output.